test/desugar/try.txt

```diff
@@ -0,0 +1,29 @@
+;;; TOOL: wast-desugar
```

Comment:
What is this test demonstrating? I guess just that you can roundtrip the try format? Might be good to add some tests that show that folding works too.